float64[] destination_1
float64 v_1
float64 a_1
bool io_0_1